nerdlets/shared/container-attributes.js

```diff
@@ -67,12 +67,12 @@ export default class ContainerAttributes extends React.Component {
   async load() {
     this.setState({ attributes: [] });
     const { account, containerId } = this.props;
-    const accountId = account.id;
+    const accountIds = account.id;
```

Comment:
This appears to just be a single account id, I don't think we want to change the naming convention

Reply:
will fix